spy/cbuild.py

```diff
@@ -222,22 +226,25 @@ def CC(self) -> list[str]:
 
     @property
     def LDFLAGS(self) -> list[str]:
-        post_js = spy.libspy.SRC.join('emscripten_post.js')
         return super().LDFLAGS + [
-            "-sEXPORTED_FUNCTIONS=['_main']",
             "-sWASM_BIGINT",
-            f"--extern-post-js={post_js}",
+            "-sERROR_ON_UNDEFINED_SYMBOLS=0",
+
         ]
 
     def c2exe(self, file_c: py.path.local, file_exe: py.path.local, *,
               opt_level: int,
               debug_symbols: bool,
               ) -> py.path.local:
-
+        post_js = spy.libspy.SRC.join('emscripten_extern_post.js')
+        EXTRA_LDFLAGS = [
+            f"--extern-post-js={post_js}",
+        ]
```

Comment:
@hoodmane I need your emscripten expertise here.
In our work together we removed these two LDFLAGS:
  - `-sEXPORTED_FUNCTIONS=['_main']"
  - `--extern-post-js={post_js}`

Now, I had to reintroduce `--extern-post-js` again in order to make `test_c2exe` to pass:
https://github.com/hoodmane/spy/blob/8b7f91c844c7f3f957c523ace21001c5293d6f4a/spy/tests/test_cbuild.py#L22-L42

The content of post_js is this:
```
export let emscriptenModule = Module();
```

In other words, this seems to be needed to tell emscripten to actually start the `main()` function when we execute the output using node.

Is it correct?